fe/fe-core/src/main/java/com/starrocks/analysis/ColumnDef.java

```diff
@@ -59,22 +57,32 @@
      */
     public static class DefaultValue {
         public boolean isSet;
-        public String value;
+        public Expr expr;
 
         public DefaultValue(boolean isSet, String value) {
```

Comment:
Why not change the definition to DefaultValue(boolean isSet, StringLiteral value) or to DefaultValue(boolean isSet, Expr value)?

Reply:
public DefaultValue(boolean isSet, String value)  is to simplify writing code。
if remove this definition it will change about 34 line position code changes were needed to write more complex code.

Reply:
I think it is better to use a more clear and simple interface.
DefalutValue can be renamed to DefaultValueDef, isSet and expr is enough.

Reply:
This will do more change, I will try.